CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v1.3.3
```

Comment:
looks like there are conflicts, you have to merge in the latest and bump the version, retag

Reply:
maybe don't tag yet until the PR is merged